### PR TITLE
TRUNK-4786: Add a Platform 2.0 is running page

### PR DESCRIPTION
**Pull request by ivange94**
https://issues.openmrs.org/browse/TRUNK-4786

### CODE REVIEW

**Comment by ivange94** [comment]
typo, there was suppose to be 'are' before the 'seeing' will correct it. Please address my comment on jira before i can do another push.

**Comment by dkayiwa** [comment]
@ivange94 i have commented on the ticket.

**Comment by dkayiwa** [comment]
@ivange94 did you see Wyclif's response at? https://github.com/openmrs/openmrs-core/commit/99e6f48ed332de05f2caf0a7654df19ee107ff02#commitcomment-14768438

File: webapp/src/main/webapp/index.htm [hunk 1]
```diff
@@ -38,7 +38,11 @@
         	<td width="100px"><div class="logo"></div></td>
         	<td>
             	<div class="bar-round">
-        			<div class="bar">OpenMRS Platform ${project.version}.${revisionNumber} Running!</div>
+        			<div class="bar">
+        				<h3>OpenMRS Platform ${project.version}.${revisionNumber} Running! </h3>
+        				<p>If you seeing this page, it means that the OpenMRS Platform is running successfully, but no user interface module is installed. To learn about the available user interfaces, see [<a href="https://wiki.openmrs.org/display/docs/Administering+Modules">https://wiki.openmrs.org/display/docs/Administering+Module/</a>]</p>
```
Comment (djazayeri) [inline webapp/src/main/webapp/index.htm]: "Administering Modules" is the wrong link here. We should create a _new_ wiki page called something like "User Interface Modules" which should describe that the OpenMRS Platform does not have a UI starting from version 2.0, and that they can install either (a) Legacy UI module, (b) Reference Application, (c) something else...

Comment (ivange94) [inline webapp/src/main/webapp/index.htm]: @dkayiwa according to what @wluyima says i can just ignore that. He also said in that commit should try running it with tomcat and i will not get that error so am testing now with tomcat, if it works fine then i will just ignore it .

Comment (ivange94) [inline webapp/src/main/webapp/index.htm]: @djazayeri I am aware that "Administering Modules" is the wrong link. I google for any openmrs wiki that was already built for that and the closest i could find was "Administering Modules". How ever i posted a comment on jira, saying that "Administering Module" link needs to be changed or the wiki modified. @djazayeri i would like to you give me all the information that you think should be on the wiki and also instructions on how to create the wiki. :-)

**Comment by ivange94** [comment]
@dkayiwa following what @wluyima said in that commit <a href="https://github.com/openmrs/openmrs-core/commit/99e6f48ed332de05f2caf0a7654df19ee107ff02#commitcomment-14768438" class="commit-link"><tt>99e6f48</tt>#commitcomment-14768438</a>, i have just ignored it since he said that is a jetty problem. However i tried testing to confirm what he said. I deployed the openmrs.war 
into tomcat and i was only getting plank pages (no output on the browser, not even errors like 404, i don't know if am doing something wrong). :-)

**Comment by dkayiwa** [comment]
@ivange94 i have merged this. Are you going to create the wiki page?

**Pull request merged**